src/components/ImageGallery/ImageGallery.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+
+// TODO: replace with import of image, rather than grabbing one off the web
+const noImagePlaceholder =
+  'https://socialistmodernism.com/wp-content/uploads/2017/07/placeholder-image.png';
+
+const CreateImageBlock = (image) => {
+  return (
+    <a
+      href={image}
+      target="_blank"
+      rel="noreferrer"
+      css={css`
+        height: 207px;
+
+        margin-right: 16px;
+        margin-bottom: 6px;
+      `}
+    >
+      <img
+        src={image}
+        alt="placeholder-text"
+        css={css`
+          width: 285px;
+          height: 207px;
```

Comment:
i noticed on mobile sized screen sizes the image goes beyond the screen a bit. we may want to add a `@media screen and (max-width: 700px)` (not sure off top of head what makes most sense for width here since we haven't done the responsive details page work). but maybe we add a note and do that work as part of that ticket?